compiler/circlechef/core/src/ModelChef.cpp

```diff
@@ -171,6 +177,12 @@ std::set<std::string> gather_customcode_set(const ::circlechef::ModelRecipe &mod
   std::set<std::string> customcode_set;
   for (const auto &operation : model_recipe.operation())
   {
+    if (operation.type() == "Custom")
+    {
+      customcode_set.insert(operation.custom_code());
+      continue;
+    }
```

Comment:
These codes can substitute below codes.
```cxx
auto op_chef = op_chef_registry().lookup(operation.type()).create(&operation);
if (op_chef->code() == circle::BuiltinOperator_CUSTOM)
  customcode_set.insert(operation.type());
```

Which means above codes should be removed.